spec/javascript/components/supplemental_application/sections/RentalAssistance.test.js

```diff
@@ -25,119 +31,216 @@ const rentalAssistance = {
   recipient: '123'
 }
 
-const rentalAssistanceInContext = (context) => (
-  <Context.Provider value={context}>
-    <Form
-      onSubmit={() => null}
-      initialValues={context.application}
-      render={({ form }) => (
-        <>
-          <form noValidate>
-            <RentalAssistance form={form} />
-          </form>
-        </>
-      )} />
-  </Context.Provider>
-)
-
 describe('RentalAssistance', () => {
-  test('should not render a table if rental assistances is empty', () => {
-    const context = cloneDeep(baseContext)
+  const getWrapper = (context) => shallowWithFormAndContext(
+    context,
+    (form) => <RentalAssistance form={form} />
+  )
 
-    const component = renderer.create(rentalAssistanceInContext(context))
+  test('matches snapshot', () => {
+    const context = cloneDeep(baseContext)
+    context.application.rental_assistances = [rentalAssistance]
+    const wrapper = getWrapper(context)
+    expect(wrapper).toMatchSnapshot()
+  })
 
-    let tree = component.toJSON()
-    expect(tree).toMatchSnapshot()
+  test('should not render a table if rental assistances is empty', () => {
+    const context = cloneDeep(baseContext)
+    const wrapper = getWrapper(context)
+    expect(wrapper.find(RentalAssistanceTable)).toHaveLength(0)
   })
 
-  test('should render a table when rental assistances are present', () => {
+  test('should render a table if rental assistances are present', () => {
     const context = cloneDeep(baseContext)
     context.application.rental_assistances = [rentalAssistance]
-
-    const component = renderer.create(rentalAssistanceInContext(context))
-
-    let tree = component.toJSON()
-    expect(tree).toMatchSnapshot()
+    const wrapper = getWrapper(context)
+    expect(wrapper.find(RentalAssistanceTable)).toHaveLength(1)
   })
 
   test('should be able to show the add rental assistance form along with the table', () => {
     const context = cloneDeep(baseContext)
     context.application.rental_assistances = [rentalAssistance]
     context.showNewRentalAssistancePanel = true
-
-    const component = renderer.create(rentalAssistanceInContext(context))
-
-    let tree = component.toJSON()
-    expect(tree).toMatchSnapshot()
+    const wrapper = getWrapper(context)
+    expect(wrapper.find(RentalAssistanceTable)).toHaveLength(1)
+    expect(wrapper.find(RentalAssistanceForm)).toHaveLength(1)
   })
+})
 
-  test('should show the Other Assistance Name in the table when Type of Assistance is Other', () => {
+describe('RentalAssistanceForm', () => {
+  const getWrapper = ({
+    assistance,
+    isNew = false,
```

Comment:
So doing this for default parameters does appear to work. I assume it would work the same in react component props